content/blog/2020-03-12-native-classroom/index.Rmarkdown

```diff
@@ -0,0 +1,93 @@
+
+Then concretely, I put the following principles into action.
+
+- *The destination is clear*: Roadmapping is central to effective teaching in any classroom. One of the biggest barriers for students are the tacit expectations teachers place on them, including why they are taking the class and their performance and behavior in it. Students should have a clear outline about what we're doing throughout and at each stage of the course: what they will learn from it, how and when their performance will be evaluated, and how it will be useful to them outside the classroom. Yup, a good written syllabus + explicit and regular expression of expectations and learning objectives are foundational for inclusive classroms. Doing this well takes time, practice, and planning. ^[[Learning Objectives and Backward Design](https://tep.uoregon.edu/learning-objectives-and-backward-design)]
+
+- *Everyone is up to speed*: At the end of any 15-20 minute lecture chunk I use low-stakes, formative assessments (e.g. ungraded quizes^[[Formative vs. Summative Assessments: What's the Difference?](https://www.aeseducation.com/blog/formative-vs.-summative-assessments-what-do-they-mean)]) to focus everyone's attention on the big picture of what we've just talked about and track/realign any misconceptions.
+
+- *Representation matters*: For slideshows, I aim to highlight Native people that are working to solve the problems showcased in each dataset. Whenever possible, I also choose pictures of native people applying the skills that the students are learning. Each of these gives students role models, so they can envision themselves doing data analysis for public health and social justice.
+
+- *Make the connection*: Upon choosing their data set and at mileposts during their analysis, I ask students to express how the data relates to them and their communities, and how they can share insights from the data with others. Letting students tell their stories builds trust. In the future, I plan to use a talking stick as each speaker shares.
+
+- *You have agency*: I let students decide on their project datasets.
+
+- *We’re a team*: Working in groups begins early. In teams of 3-4, students alternate roles as “lead investigator”, “coder”, and “reporter”. The teams take time to communicate as a class via their reporters.
+
+- *Tell me more*: I always end with a survey (I use SurveyMonkey). This gives students a clear chance to let me know what datasets spoke to them, what tripped them up, and what types of data they’d be interested to work with in the future (as well as the various technical skills they’ve gained).
+
+
+# **2. The importance of logistics and technology**
+
+Attitudes are vital. Unfortunately, they can't do much if technical difficulties completely hamper students' engagement with material. The students we seek to recruit and empower in STEM will often be in resource-limited institutions and this will require additional tricks and strategies from instructors.
```

Comment:
"Unfortunately, students can't do more if technical difficulties hamper their engagement with material." (I.e., put "students" at the start and use "their" later so that the referent is clear.)